sdk/core/azure-core/test/ut/CMakeLists.txt

```diff
@@ -59,6 +59,10 @@ target_include_directories (azure-core-test PRIVATE $<BUILD_INTERFACE:${CMAKE_CU
 
 target_link_libraries(azure-core-test PRIVATE azure-core gtest gmock)
 
+if (MSVC)
+  target_compile_options(azure-core-test PUBLIC /wd26495 /wd26812)
```

Comment:
Ditto from above.  Its good to have comments about the specific warnings we turn on/off.  Helps users to quickly understand what our build system is doing.